Website/src/rendering/src/components/Products/ProductDetailsContent.tsx

```diff
@@ -148,14 +150,34 @@ const ProductDetailsContent = ({
     async (e: FormEvent) => {
       e.preventDefault();
       setLoading(true);
-      await dispatch(
+      const response = await dispatch(
         createLineItem({
           ProductID: product.ID,
           Quantity: quantity,
           Specs: specValues,
         })
       );
       setLoading(false);
+
+      const resPayload: { LineItems?: LineItem[] } = response?.payload;
+      const lineItem = resPayload?.LineItems[resPayload.LineItems.length - 1];
+      const addToCartPayload: Partial<AddToCartPayload> = {
+        // TODO change when possible to select language from dropdown
+        language: 'EN',
+        currency: product.xp.Currency,
+        product: {
+          type: product.xp.ProductType.toUpperCase(),
+          item_id: lineItem.Variant?.ID || lineItem.ProductID,
+          name: lineItem.Product.Name,
+          orderedAt: lineItem.DateAdded,
```

Comment:
Is the DateAdded the date when the first of this product was added to the cart or the date when the current "add to cart" click happened? We should log the current operation date.